doc/util/gen-package-list.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+import base64
+import requests
+from pprint import pprint
+import re
+
+# get authentication token for GitHub
+secret = open("secret-token.txt", "r+")
```

Comment:
Are we guaranteed to hit a request limit if we don't have this? I wonder if we should have a way to omit using authentication. Curious on your thoughts, but I don't see it as a necessary change for this first draft.

Reply:
Not guarenteed at all but I wanted to put that in there just in case someone ever did get the authentication token for Chapel.

Reply:
OK, so adding an option to skip this could be helpful in the future -- would you mind adding a TODO comment for that?